.github/workflows/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "hourly"
```

```suggestion
    schedule:
      interval: "weekly"
    reviewers:
      - "customerio/mobile"
```

How about automatically adding mobile group as reviewers?